File: app/src/main/java/ru/ip_fateev/lavka/Inventory/Product.java
package ru.ip_fateev.lavka.Inventory;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

@Entity
public class Product implements Serializable {

    @PrimaryKey
    public long id;

    public String name;

    public Double price;

    public String barcode;

    public String getId() {
        return id + "";
    }

    public String getName() {
        return name;
    }

    public String getPrice() {
        return price + "";
    }

    public String getBarcode() {
        return barcode;
    }
}

File: app/src/main/java/ru/ip_fateev/lavka/Inventory/Product.kt
package ru.ip_fateev.lavka.Inventory

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Product (
    @PrimaryKey
    val id: Long,
    val name: String,
    val price: Double,
    val barcode: String,
) : java.io.Serializable